flatflow/scheduler/internal/algorithm/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 The FlatFlow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if(BUILD_TESTING)
+  add_executable(
+    algorithm_test
+    regression_test.cc
```

Comment:
There seems to have been some problems during upstream merge.
This file should contain exactly one line of changes.